tests/gtest/aviftest_helpers.h

```diff
@@ -66,13 +66,15 @@ struct AvifIOLimitedReader {
   static constexpr uint64_t kNoClamp = std::numeric_limits<uint64_t>::max();
 
   avifIO io;
-  avifIO* underlayIO;
+  avifIO* underlyingIO;
```

Comment:
I renamed this member "underlyingIO". I think "underlying" is more correct than "underlay". Please let me know if I misunderstood the meaning.

Other alternatives include "lowerIO", "lowerLayer", "wrappedIO".

Reply:
Yes this is exactly what I mean. I couldn't remember this very word when writing the code, so used "underlay" instead which feels similar.